src/Sarif.UnitTests/TestData/InsertOptionalDataVisitor/CoreTests.sarif

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
+  "version": "2.0.0",
```

Comment:
Can we --pretty-print these files? They're a bit hard to read with so much nesting and multiple properties on a single line.